tests/API/Management/EmailTemplatesTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+namespace Auth0\Tests\API\Management;
+
+use Auth0\SDK\API\Management;
+use Auth0\Tests\API\ApiTests;
+use GuzzleHttp\Exception\ClientException;
+
+/**
+ * Class EmailTemplateTest
+ *
+ * @package Auth0\Tests\API\Management
+ */
+class EmailTemplateTest extends ApiTests
+{
+
+    const EMAIL_TEMPLATE_NAME = 'enrollment_email';
+
+    /**
+     * Management API token with scopes read:email_templates, create:email_templates, update:email_templates
```

Comment:
while this is true for calling all endpoints, some of them can be called with only 1 of this scopes. So this is not exactly true

Reply:
This is in a test and for the tests here to run, it needs all three.